Source/Fuse.Elements/Resources/Exif/Tests/OrientationTest/Exif.OrientationTest.uno

```diff
@@ -0,0 +1,104 @@
+using Uno.Testing;
+using Fuse.Resources.Exif;
+
+public class OrientationTest
+{
+	public void AssertImageOrientation(byte[] bytes, ImageOrientation expected)
+	{
+		var ori = ExifData.FromByteArray(bytes).Orientation;
+		Assert.AreEqual(expected, ori);
+	}
```

Comment:
First off, please make `expected` the first argument here, this is what all other Assert-methods do.

Secondly, if this fails, it's going to fail with the location being inside this call, which isn't particularly helpful. If you add `[CallerFilePath] string filePath = "", [CallerLineNumber] int lineNumber = 0, [CallerMemberName] string memberName = ""` to the signature, and pass these to Assert.AreEqual as well, the correct location should be blamed instead.

Third, it seems to me like all call-sites call BundleFile.ReadAllBytes() as the input here. Maybe we could just make this helper take BundleFile as an argument instead? If so, the call-sites would read:

```uno
AssertImageOrientation(ImageOrientation.Identity, import("bird0.jpg"));
```
...which I think is pretty neat!

Lastly (and this is extremely unimportant at this point), but this method could also be marked as static ;)